common/actions.js

```diff
@@ -347,7 +347,7 @@ export const getActivity = async (data) => {
 };
 
 export const getZipFilePaths = async (data) => {
-  return await returnJSON(`api/zip/get-paths`, {
+  return await returnJSON(`../api/zip/get-paths`, {
```

Comment:
How is this possible?

Reply:
https://denmchenry.com/posts/relative-urls/, actually the browser interprets it as the parent URL, I think why it didn't work previously is because it didn't have `/` before `api`, I thought it did have the `/` that's why I added the double dots. I'll update it right now